packages/shared/src/components/referral/InviteLinkInput.tsx

```diff
@@ -29,20 +36,25 @@ export function InviteLinkInput({
       event_name: AnalyticsEvent.CopyReferralLink,
       target_id: targetId,
     });
+    setHasCopied(true);
 
     if (onCopy) {
       onCopy();
     }
   };
 
   const renderText = () => {
-    const copy = 'Copy link';
+    const copy = text?.initial ?? 'Copy link';
+
+    if (text?.copying) {
+      return copying ? text?.copying : copy;
+    }
```

Comment:
Where did this come from, I don't see this part in the miro

Reply:
When click a "Copied" button, nothing really changed, kind of a bad UX, but I can remove this if we don't want it.

Reply:
Rather check this UX with Sab, I think they are not aware of how it works maybe.

Reply:
Or they intended to have:
Copy link -> click -> Copied -> return back to Copy link 
As it is in current behaviour

Reply:
Ah, I see now why there is a short delay with returning it back. In that case, the previous one was correct.

Reply:
Fixed with the copied state now 🚀